runtime/development/src/lib.rs

```diff
@@ -922,7 +922,7 @@ impl pallet_treasury::Config for Runtime {
 	type Burn = Burn;
 	// we burn and dont handle the unbalance
 	type BurnDestination = ();
-	type Currency = Tokens;
```

Comment:
BTW, It's not a breaking change, and we align all runtimes with this.

Reply:
Why? We should use Tokens everywhere in order to keep restrictions on place.

Reply:
Then we should change instead from `Balance` to `Tokens` in `altair` and `centrifuge`

Reply:
Done